installer/lib/kitto_new.ex

```diff
@@ -48,6 +50,7 @@ defmodule Mix.Tasks.Kitto.New do
     {:text, "new/assets/stylesheets/application.scss", "assets/stylesheets/application.scss"},
     {:keep, "new/public/assets",                       "public/assets"},
     {:text, "new/public/assets/favicon.ico",           "public/assets/favicon.ico"},
+    {:text, "new/public/assets/images/nyan.gif",       "public/assets/images/nyan.gif"},
```

Comment:
What is the license of this image? I know that it's too common on the internet but it may be violating copyrights :-/ and cause issues to the project.